FXyz-Importers/src/main/java/org/fxyz3d/importers/maya/MayaImporter.java

```diff
@@ -32,134 +32,103 @@
 
     @Override
     public Model3D load(URL url) throws IOException {
-        load(url, false);
-        return new Model3D() {
-            @Override
-            public Optional<Timeline> getTimeline() {
-                return Optional.of(MayaImporter.this.timeline);
-            }
-
-            @Override
-            public Group getRoot() {
-                return MayaImporter.this.root;
-            }
-        };
+        return load(url, false);
     }
 
     @Override
     public Model3D loadAsPoly(URL url) throws IOException {
-        load(url, true);
-        return new Model3D() {
-            @Override
-            public Optional<Timeline> getTimeline() {
-                return Optional.of(MayaImporter.this.timeline);
-            }
-
-            @Override
-            public Group getRoot() {
-                return MayaImporter.this.root;
-            }
-        };
+        return load(url, true);
     }
 
-    //=========================================================================
-    // MayaImporter.load
-    //=========================================================================
-
-    private void load(URL url, boolean asPolygonMesh) {
-        try {
-            Loader loader = new Loader();
-            loader.load(url, asPolygonMesh);
-
-            // This root is not automatically added to the scene.
-            // It needs to be added by the user of MayaImporter.
-            //            root = new Xform();
-
-            // Add top level nodes to the root
-            int nodeCount = 0;
-            for (Node n : loader.loaded.values()) {
-                if (n != null) {
-                    // Only add a node if it has no parents, ie. top level node
-                    if (n.getParent() == null) {
-                        if (Loader.DEBUG) {
-                            System.out.println("Adding top level node " + n.getId() + " to root!");
-                        }
-                        n.setDepthTest(DepthTest.ENABLE);
-                        if (!(n instanceof MeshView) || ((TriangleMesh)((MeshView)n).getMesh()).getPoints().size() > 0) {
-                            root.getChildren().add(n);
-                        }
+    private Model3D load(URL url, boolean asPolygonMesh) {
+        Loader loader = new Loader();
+        loader.load(url, asPolygonMesh);
+
+        // This root is not automatically added to the scene.
+        // It needs to be added by the user of MayaImporter.
+        //            root = new Xform();
+
+        MayaGroup mayaRoot = new MayaGroup();
+        // Add top level nodes to the root
+        int nodeCount = 0;
+        for (Node n : loader.loaded.values()) {
+            if (n != null) {
+                // Only add a node if it has no parents, ie. top level node
+                if (n.getParent() == null) {
+                    if (Loader.DEBUG) {
+                        System.out.println("Adding top level node " + n.getId() + " to root!");
```

Comment:
Can we use a Logger instead? Or at least factorize it like in `ObjImporter`

Reply:
I'm happy to change this to logger, all these debug statements seem to be all over the place